xarray/core/accessor_dt.py

```diff
@@ -16,9 +16,21 @@
 def _season_from_months(months):
     """Compute season (DJF, MAM, JJA, SON) from month ordinal"""
     # TODO: Move "season" accessor upstream into pandas
-    seasons = np.array(["DJF", "MAM", "JJA", "SON"])
+    seasons = np.array(["DJF", "MAM", "JJA", "SON", np.nan])
```

Comment:
If we want to return a true np.nan among the strings, it would be via a simple list, right ?
Which would be strange since xarray doesn't deal with lists usually ?